heudiconv/convert.py

```diff
@@ -324,10 +324,11 @@ def convert_dicom(item_dicoms, bids, prefix,
             outfile = op.join(dicomdir, op.basename(filename))
             if not op.islink(outfile):
                 # TODO: add option to enable hardlink?
-                if symlink:
-                    os.symlink(filename, outfile)
-                else:
-                    os.link(filename, outfile)
+#                if symlink:
+#                    os.symlink(filename, outfile)
+#                else:
+#                    os.link(filename, outfile)
+                shutil.copyfile(filename, outfile)
```

Comment:
that would lead to huge performance impact on any sizeable dataset for no good reason really.  The fix here would be just to get a full path to the filename, not a relative path (I think that is what was broken, wasn't it?)

Reply:
oh, I misunderstood what issue we are solving.. #120 concerns about broken symlinks to dicom outputs, while I thought we are talking about failing "embedding" (works in my elderly version without a message at least)

Reply:
Yes, the link option was not working for me. On my system my dicom tar-ball is extracted to /tmp/..., and the conversion created absolute links in <outdir>/sub/func/... to this /tmp dir, but at the end the tmp dir is removed,my dicom files are gone, and I have only links to nowhere.